kalite/updates/static/js/updates/bundle_modules/update_videos.js

```diff
@@ -415,10 +415,12 @@ function getSelectedStartedMetadata(data_type) {
 }
 
 function setNodeClass(node, className) {
-    if (node.extraClasses != className) {
-        node.extraClasses = className;
-        if (node.parent !== null) {
-            updateNodeClass(node.parent);
+    if (node !== null) {
```

Comment:
> This check could happen before we get to this function call, but this seemed the neatest place to check.
 
That's not consistent with `updateNodeClass`, where a similar check is made before calling the function: https://github.com/rtibbles/ka-lite/blob/prevent_type_error/kalite/updates/static/js/updates/bundle_modules/update_videos.js#L421

Same with `recurNodeParent`: https://github.com/rtibbles/ka-lite/blob/prevent_type_error/kalite/updates/static/js/updates/bundle_modules/update_videos.js#L486

Do you feel that affects the future readability of this module? If not, I'll merge. Otherwise, I'd say let's establish the convention that these functions *must* take a non-null node (or conversely add similar null-guards to the other functions which have node parameters).